lib/app/Readme.md

```diff
@@ -8,12 +8,20 @@ In your configuration, you can enable an app using something like this:
 
 ```yaml
 apps:
-    # Replace appName with the name of the app you wish to enable
-    appName:
-        responseCache: 180
-        disabled: false
+  # Replace appName with the name of the app you wish to enable
+  appName:
+    responseCache: 180
+    disabled: false
+    sharedData:
```

Comment:
Wow i didn't even know that existed, however i can see its merits
After a chat with @ronkorving, I agree that since we use external build systems now that can inject extra configuration, we no longer need something like this.